DataFormats/HGCalDigi/interface/HGCalFEDPacketInfoSoA.h

```diff
@@ -0,0 +1,66 @@
+#ifndef DataFormats_HGCalDigi_interface_HGCalFEDPacketInfoSoA_h
+#define DataFormats_HGCalDigi_interface_HGCalFEDPacketInfoSoA_h
+
+#include <cstdint>  // for uint8_t
+
+#include <Eigen/Core>
+
+#include "DataFormats/SoATemplate/interface/SoACommon.h"
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/SoATemplate/interface/SoAView.h"
+
+namespace hgcaldigi {
+
+  namespace FEDUnpackingFlags {
```

Comment:
rather than constants in a namespace, would it make sense to define this as `enum class FEDUnpackingFlags : uint8_t { ... }` ?

Reply:
(unless it makes the rest of the code more complicated)

Reply:
can work, it requires some static_cast for bitwise operation

Reply:
I'd argue `constexpr` variables to be more idiomatic for number constants than an enumeration.

Reply:
indeed the code gets a bit clunky later one with all the static_cast needed if I turn it to enum class. maybe we leave it as is for the moment. We also used `constexpr` variables for other SoA in this package